modules/shared/src/main/scala/org/tessellation/security/signature/SignedValidator.scala

```diff
@@ -105,6 +105,23 @@ object SignedValidator {
           .toNes
       }.map(SignersNotInSeedlist).toInvalidNec(signed)
 
+    def validateSignedByMajority[A](seedlist: Option[Set[PeerId]], signed: Signed[A]): SignedValidationErrorOr[Signed[A]] =
+      seedlist match {
+        case None => signed.validNec
+        case Some(peers) =>
+          val minRequired = peers.size / 2 + 1
+          val signingPeers = signed.proofs
+            .map(_.id.toPeerId)
+            .toSortedSet
```

Comment:
do we need the set to be sorted?

Reply:
@marcinwadon  we aren't sorting; we're unwrapping the NonEmptySet so I can intersect it with the peer set.

Reply:
ok :+1: